.github/workflows/prod-api-CD.yml

```
@@ -0,0 +1,52 @@
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+
+      - name: Inject API secrets
+        run: |
+          echo "${{ secrets.API_APPLICATION_PROD }}" > api-server/src/main/resources/application-api-prod.yml
+          mkdir -p api-server/src/main/resources/firebase
+          echo "${{ secrets.API_FIREBASE_JSON }}" > api-server/src/main/resources/firebase/napzak-firebase-adminsdk.json
+
+      - uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)